ci/checks/doxygen.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION.
+###############################
+# cuDF doxygen warnings check #
+###############################
+
+# skip if doxygen is not installed
+if ! [ -x "$(command -v doxygen)" ]; then
+  echo -e "warning: doxygen is not installed"
+  exit 0
+fi
+
+# Utility to return version as number for comparison
+function version { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+# doxygen supported version 1.8.20 to 1.9.1
+DOXYGEN_VERSION=`doxygen --version`
```

Comment:
```suggestion
DOXYGEN_VERSION=$(doxygen --version)
```

For consistency with the rest of the script, we should switch this to use `$()` instead of ``` `` ```.